schemas/threat.yml

```diff
@@ -111,18 +111,9 @@
       short: Indicator display name
       description: >
         The display name indicator in an UI friendly format
-      expected_values:
```

Comment:
This looks like it should have been an example. Does the schema allow this to be converted into an `example` (not sure if a list of individual examples can be conveyed)?

Reply:
No, the schema does not. I thought about migrating the values to `example` originally but didn't want to infer the fields expected an array of values.

In an earlier revision, I moved the examples into `description`. But after moving, I liked eliminating the specific values in favor of generic terms like URL, port, IP, etc.